apps/web/components/v2/eventtype/EventAdvancedTab.tsx

```diff
@@ -274,7 +274,6 @@ export const EventAdvancedTab = ({ eventType, team }: Pick<EventTypeSetupInfered
       <Controller
         name="successRedirectUrl"
         control={formMethods.control}
-        defaultValue={hashedUrl}
```

Comment:
The input below already have a defaultValue using successRedirectUrl, so this didn't seem necessary. So, instead of changing the variable I simply removed it.